test/unit/api/ingest/_data.js

```diff
@@ -63,72 +63,72 @@ define(function (require) {
-            expect(bulkResponse.body[0]).to.have.property('created');
-            expect(bulkResponse.body[0].created).to.be(100);
+          .then((dataResponse) => {
+            expect(dataResponse.body[0]).to.have.property('created');
+            expect(dataResponse.body[0].created).to.be(100);
           });
         });
 
         bdd.it('should report any indexing errors per document under an "errors.index" key', function () {
-          return request.post('/kibana/names/_bulk')
+          return request.post('/kibana/names/_data')
             .attach('csv', 'test/unit/data/fake_names_with_mapping_errors.csv')
             .expect(200)
-            .then((bulkResponse) => {
-              expect(bulkResponse.body[0]).to.have.property('created');
-              expect(bulkResponse.body[0].created).to.be(98);
-              expect(bulkResponse.body[0]).to.have.property('errors');
-              expect(bulkResponse.body[0].errors).to.have.property('index');
-              expect(bulkResponse.body[0].errors.index.length).to.be(2);
+            .then((dataResponse) => {
+              expect(dataResponse.body[0]).to.have.property('created');
+              expect(dataResponse.body[0].created).to.be(98);
+              expect(dataResponse.body[0]).to.have.property('errors');
+              expect(dataResponse.body[0].errors).to.have.property('index');
+              expect(dataResponse.body[0].errors.index.length).to.be(2);
             });
         });
 
         bdd.it('should report any csv parsing errors under an "errors.other" key', function () {
-          return request.post('/kibana/names/_bulk')
+          return request.post('/kibana/names/_data')
             .attach('csv', 'test/unit/data/fake_names_with_parse_errors.csv')
             .expect(200)
-            .then((bulkResponse) => {
+            .then((dataResponse) => {
               // parse errors immediately abort indexing
-              expect(bulkResponse.body[0]).to.have.property('created');
-              expect(bulkResponse.body[0].created).to.be(0);
+              expect(dataResponse.body[0]).to.have.property('created');
+              expect(dataResponse.body[0].created).to.be(0);
 
-              expect(bulkResponse.body[0]).to.have.property('errors');
-              expect(bulkResponse.body[0].errors).to.have.property('other');
-              expect(bulkResponse.body[0].errors.other.length).to.be(1);
+              expect(dataResponse.body[0]).to.have.property('errors');
+              expect(dataResponse.body[0].errors).to.have.property('other');
+              expect(dataResponse.body[0].errors.other.length).to.be(1);
             });
         });
 
       });
 
       bdd.describe('optional parameters', function () {
         bdd.it('should accept a custom delimiter query string param for parsing the CSV', function () {
-          return request.post('/kibana/names/_bulk?delimiter=|')
+          return request.post('/kibana/names/_data?delimiter=|')
```

Comment:
Just noticed this. The `|` could be passed in as `%7C` because of URL encoding. Mind adding a test to make sure that works?